.github/workflows/build-deploy-microservices.yml

```diff
@@ -0,0 +1,246 @@
+name: Build & Deploy Microservices
+
+on:
+  pull_request:
+    paths-ignore:
+    - 'README.md'
+    - 'CODE_OF_CONDUCT.md'
+    - 'CONTRIBUTING.md'
+    - 'LICENSE'
+    - 'SECURITY.md'
+    - 'docs/**'
+    - '.github/**'
+    - '!.github/workflows/build-deploy.yml'
```

Comment:
I guess it would be a good idea to update this path to `!.github/workflows/build-deploy-microservices.yml `